neard/tests/load_genesis.rs

```diff
@@ -0,0 +1,9 @@
+use near_chain_configs::Genesis;
+
+#[test]
+fn test_load_genesis() {
```

Comment:
wouldn't this test fail if we enable `protocol_feature_add_account_versions`?

Reply:
Why will it? And I think it should've ran as part of CI, no?

Reply:
I misread and thought we are using the same file in both cases :)

Reply:
Should the node with the feature enabled be able to parse the existing genesis file?